google/cloud/storage/internal/grpc_bucket_request_parser.h

```diff
@@ -38,6 +38,8 @@ struct GrpcBucketRequestParser {
 
   static StatusOr<google::storage::v2::UpdateBucketRequest> ToProto(
       PatchBucketRequest const& request);
+  static StatusOr<google::storage::v2::UpdateBucketRequest> ToProto(
```

Comment:
this can not return a bad status. do we want this signature?
```cc
google::storage::v2::UpdateBucketRequest ToProto(UpdateBucketRequest const& request);
```